docs-core/src/main/java/com/sismics/docs/core/model/jpa/Review.java

```diff
@@ -0,0 +1,102 @@
+package com.sismics.docs.core.model.jpa;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * Link between an user and a group.
+ * 
+ * @author jessedin
+ */
+@Entity
+@Table(name = "T_REVIEW")
+public class Review {
+
+    /**
+     * User ID.
+     */
+    @Id
+    @Column(name = "REV_IDUSER_C", length = 36)
+    private String userId;
```

Comment:
An open question: do we want to show the user's names or IDs? I don't have a good answer. In the meantime, just to hedge our bets, can we add the names also to the table?

In a real DB, we'd just call INNER JOIN on the User table toobtain their names, but we are assuming for the purposes of this feature that all of the users in the table only exist in the this table (i.e. we have not added them to the User table).